src/systemcmds/tests/tests_main.c

```diff
@@ -93,7 +93,9 @@ const struct {
 	/* external tests */
 	{"commander",		commander_tests_main,	0},
 	{"controllib",		controllib_test_main,	0},
+#ifndef __PX4_NUTTX
```

Comment:
@dagar do you remember why?

Reply:
I believe it was the mavlink ftp tests that were deleting generated test material. I'd like to bring them back if anyone has time for a closer look.

Reply:
I'm doing it in #12976.